nilearn/plotting/img_plotting.py

```diff
@@ -312,6 +327,155 @@ def _load_anat(anat_img=MNI152TEMPLATE, dim=False, black_bg='auto'):
+            figure is used.
+        title : string, optional
+            The title displayed on the figure.
+        annotate: boolean, optional
+            If annotate is True, positions and left/right annotation
+            are added to the plot.
+        draw_cross: boolean, optional
+            If draw_cross is True, a cross is drawn on the plot to
+            indicate the cut plosition.
+        black_bg: boolean, optional
+            If True, the background of the image is set to be black. If
+            you wish to save figures with a black background, you
+            will need to pass "facecolor='k', edgecolor='k'" to pylab's
+            savefig.
+        cmap: matplotlib colormap, optional
+            The colormap for the atlas maps 
+        vmin: float
+            Lower bound for plotting, passed to matplotlib.pyplot.imshow
+        vmax: float
+            Upper bound for plotting, passed to matplotlib.pyplot.imshow
+
+    """
+    anat_img, black_bg, _, _ = _load_anat(anat_img, dim=dim,
+                                          black_bg=black_bg)
+  
+    display = plot_img(anat_img, cut_coords=cut_coords,
+                       output_file=output_file, display_mode=display_mode,
+                       figure=figure, axes=axes, title=title,
+                       threshold=None, annotate=annotate,
+                       draw_cross=draw_cross, black_bg=black_bg,
+                       cmap=plt.cm.gray, **kwargs)
+ 
+    maps_img = _utils.check_niimg_4d(maps_img)
+    n_maps = maps_img.shape[3]
+   
+    if isinstance(cmap, basestring):
+        color_map = plt.cm.get_cmap(cmap)
+
+    # Build a custom colormap for displaying contours
+    color_list = color_map(np.linspace(0, 1, n_maps))
+    
+    if threshold is None:
+        # it will use default percentage, 
+        # for a nicer look avoiding maximum overlaps for visualization
+        threshold = ["99.7%"] * n_maps
+     
+    if hasattr(threshold, '__iter__'):
+        if len(threshold) != n_maps:
+            raise TypeError('The list of values to threshold '
+                                'should be equal to number of maps')
+    else:
+        threshold = [threshold] * n_maps 
+    
+    for i, (map_img, color, thr) in enumerate(zip(iter_img(maps_img), color_list, threshold)):
+        # To threshold or choose the level of the contours
+        data = map_img.get_data()
+        if isinstance(thr, basestring):
+            message = ("If 'threshold' is given as a string it "
+                       'should be a number followed by the percent sign, '
+                       'e.g. "25.3%"')
+            if not thr.endswith('%'):
+                raise ValueError(message)
+            try:
+                percentile = float(thr[:-1])
+            except ValueError as exc:
+                exc.args += (message, )
+                raise
+            thr = fast_abs_percentile(data, percentile) + 1e-4
```

Comment:
When you add magic values, specify why you do that. Plus, this value is not small.